Mathlib/Algebra/Module/Submodule/Map.lean

```diff
@@ -619,3 +619,25 @@ def compatibleMaps : Submodule R (N →ₗ[R] N₂) where
 #align submodule.compatible_maps Submodule.compatibleMaps
 
 end Submodule
+
+namespace LinearMap
+
+variable { M M' R : Type* } [Semiring R] [AddCommMonoid M] [AddCommMonoid M']
+  [Module R M] [Module R M']
+
+/-- A linear map between two modules restricts to a linear map from any submodule p of the
+domain onto the image of that submodule.
+
+This is the linear version of AddMonoidHom.addSubmonoidMap and AddMonoidHom.addSubgroupMap.-/
```

Comment:
Thanks for these cross-references, they're very useful to have in docstrings.

If you put them in backticks, docgen will link them:
```suggestion
This is the linear version of `AddMonoidHom.addSubmonoidMap` and `AddMonoidHom.addSubgroupMap`. -/
```